FlorenceBML/BMLacq306/BMLacq306.xml

```diff
@@ -838,6 +851,7 @@ schematypens="http://relaxng.org/ns/structure/1.0"
             <change when="2023-02-28" who="CH">updated msParts</change>
             <change when="2023-03-01" who="CH">updated msParts</change>
             <change when="2023-03-02" who="CH">updated msParts and additions</change>
+            <change when="2023-03-15" who="CH">Identification of scribe, owner insitution</change>
```

Comment:
I guess `who = "DN"`
it probably is good to also add a `<provenance>` in `<history>` , thank you!

Reply:
I did, didn't I (for the entire volume)?